### DISCUSSION TITLE
Default system language

### DISCUSSION THREAD

**Discussion by nedevski** [mealie-recipes/mealie#2853] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I have a lot of devices and every time I log in, Mealie is in English and I have to change it. Not that big of an issue for me, but it kinda is for my mom and dad.

### Please provide a concise description of the feature that would resolve your issue.

Default global system locale setting and/or the preferred user locale to be saved in the database and to be configurable on user creation.

### Please consider and list out some caveats or tradeoffs made in your design decision

Additional data in the DB which I think won't be that big of a deal.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by boc-the-git** [comment]
I can't speak to the complexity of implementing it, but this seems a reasonable enough request I think. An environment variable that sets a default language, that can still be overridden by user settings.
Gut instinct says it's probably not that complicated to implement.
